src/gameManager.js

```diff
@@ -1,197 +1,248 @@
-import { mobileControls } from './touch.js';
-import { rocket } from './rocket.js';
-import { airfighter } from "./airfighter.js";
-import { deleteEnemies, createEnemies } from './enemy.js';
-import { soundRocketShot, soundMainTheme, soundIntro } from "./music.js";
-import { Step, gameState, fps } from './step.js';
-import { fireGatlingEnemy } from './gatling.js';
-import { resetAmmo } from './ammo.js';
-
-let isGameStarted = false;
-let introduction = document.querySelector('#introduction');
-introduction.addEventListener('click', introductionSkip);
+import { mobileControls } from './touch.js'
+import { rocket } from './rocket.js'
+import { airfighter } from './airfighter.js'
+import { deleteEnemies, createEnemies, enemies } from './enemy.js'
+import {
+  soundRocketShot,
+  soundMainTheme,
+  soundIntro,
+  soundLevelComplete,
+  soundboss,
+} from './music.js'
+import { Step, gameState, fps } from './step.js'
+import { fireGatlingEnemy } from './gatling.js'
+import { resetAmmo } from './ammo.js'
+import { level2boss, createBoss } from './Boss.js'
+import { bossPopup } from './rendering/Helpers.js'
+
+let isGameStarted = false
+let introduction = document.querySelector('#introduction')
+introduction.addEventListener('click', introductionSkip)
 
 const levelState = {
-  isLevelFinished : false,
-  levelNumber : 1,
+  isLevelFinished: false,
+  levelNumber: 0,
 }
 
 export function startGame() {
-  console.log('startGame');
-  initKeybordMovement();
+  console.log(`startGame, level :${levelState.levelNumber}`)
+  initKeybordMovement()
   if (levelState.levelNumber == 1) {
     startLevel1()
   }
   if (levelState.levelNumber == 2) {
     startLevel2()
   }
+  if (levelState.levelNumber == 3) {
+    startLevel3()
+  }
+}
+
+function changeLevel() {
+  isGameStarted = false
+  soundLevelComplete.pause()
+  document
+    .querySelector('#levelComplete')
+    .addEventListener('click', (event) => {
+      introduction.style.display = 'block'
+      introduction.style.zIndex = levelState.levelNumber * 2
```

Comment:
Please use same z-index for all levels.
I don't think that `z-index` should depend on `levelNumber`.